app/controllers/acuant_sdk_controller.rb

```diff
@@ -0,0 +1,44 @@
+class AcuantSdkController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  ACUANT_SDK_STATIC_FILES = %w[
+    AcuantImageProcessingService.wasm
+    AcuantImageProcessingWorker.min.js
+    AcuantImageProcessingWorker.wasm
+    AcuantJavascriptWebSdk.min.js
+  ].freeze
+
+  def show
+    # Only render files on an allowlist to prevent path traversal issues
+    render plain: 'Not found', status: :not_found unless requested_asset_permitted?
+    send_data(
```

Comment:
I think that send_file can be faster because it doesn't require having the data in memory already, and takes the same options:

https://api.rubyonrails.org/v5.2.3/classes/ActionController/DataStreaming.html

Reply:
Good call, latest commit has that change